package.json

```diff
@@ -63,7 +63,7 @@
     "generate:blessed": "node ./tools/generate_png_images.js ../releases ./build/images/blessed",
     "generate": "npm run generate:current && npm run generate:blessed",
     "diff": "./tools/visual_regression.sh",
-    "test": "npm run lint && npm run qunit && npm run generate && npm run diff"
+    "test": "npm run start && npm run generate && npm run diff"
```

Comment:
I'm not sure this is a way to go, since `npm start` will copy files from build to release it was supposed to be checked in. By doing this you are not longer forcing to check in releases.

Reply:
I would suggest just checkin the releases on the formal tagged builds.

Reply:
In order to achieve that we have to produce the releases to pass the CI tests. @0xfe  what do you think?

Reply:
should we do it now in this PR? It is about to remove babel

Reply:
@0xfe you were concerned about making sure that the visual tests were passing. I have realised that there are merges where the tests were not running ie.: 1924b2cc268de33fb19bbc8ba62c59b635614ea4

Now we have the following actions:
`'clean', 'eslint', 'webpack:buildDev', 'webpack:build', 'concat', 'docco', 'qunit', 'copy:release', 'generate', 'diff'`

in comparison with:
`'clean', 'webpack:buildDev', 'concat', 'qunit'`

with this approach we also get the eslint warnings displayed in the code:
![image](https://user-images.githubusercontent.com/22865285/112381760-405d6d80-8ceb-11eb-9345-4317bd3facbf.png)

@0xfe do you like it or you prefer me to revert the error as suggested by @wassertim? With your answer I will proceed and resolve this topic. I can also remove some of the steps if you prefer.

Reply:
I don't recommend changing this line either -- or we're making too many changes. Let's set `class-methods-use-this` to `off` for now.

Reply:
Actually, this is only a problem in a small number of files. Let's instead add a comment to the top of `music.ts` disabling the rule just for the file.

Reply:
> @0xfe you were concerned about making sure that the visual tests were passing. I have realised that there are merges where the tests were not running ie.: [1924b2c](https://github.com/0xfe/vexflow/commit/1924b2cc268de33fb19bbc8ba62c59b635614ea4)
> 
> Now we have the following actions:
> `'clean', 'eslint', 'webpack:buildDev', 'webpack:build', 'concat', 'docco', 'qunit', 'copy:release', 'generate', 'diff'`
> 
> in comparison with:
> `'clean', 'webpack:buildDev', 'concat', 'qunit'`
> 
@0xfe what about the change in package.json?

Reply:
@wassertim hightlighted that this alreay answer "I don't recommend changing this line either -- or we're making too many changes."
I will make the comment in music.ts roll back package and checking the releases dir